model/aero/aero_final_analysis.yaml.j2

```diff
@@ -0,0 +1,18 @@
+geometry:
+  fms initialization:
+    namelist filename: "{{aero_fv3jedi_files_path}}/fmsmpp.nml"
+    field table filename: "{{aero_fv3jedi_files_path}}/field_table"
+  akbk: "{{aero_fv3jedi_files_path}}/akbk.nc4"
+  layout:
+  - {{aero_layout_x}}
+  - {{aero_layout_y}}
+  npx: {{aero_npx_anl}}
+  npy: {{aero_npy_anl}}
+  npz: {{aero_npz_anl}}
+  field metadata override: "{{aero_fv3jedi_files_path}}/fv3jedi_fieldmetadata_restart.yaml"
+datapath: {{ aero_analysis_path }}
```

Comment:
Does this section need to be in an "output" section like the other yaml in this PR has?

Reply:
no that's a JEDI quirk, but I'm worried it's not working properly on the geometry/resolution... so this PR may be largely moot, still testing

Reply:
ok it seems this doesn't work, so I guess I'll need to run a separate executable, that's a problem for a later time, so this PR becomes a 3 line PR